src/rail/estimation/algos/deepdisc.py

```diff
@@ -23,7 +25,7 @@
 
 from rail.deepdisc.configs import *
 
-
+#! I don't think this class is used any more - confirm with Grant
 class DeepDiscDictInformer(CatInformer):
```

Comment:
Verify if this class is actually going to be used. If so, we should include the caching code here.

Reply:
This can be deleted, it was a holdover from early experimenting